test/jdk/TEST.groups

```diff
@@ -20,6 +20,14 @@
 #  questions.
 #
 
+###############################################################################
+#
+# Catch-all definitions
+#
+
+all = \
```

Comment:
Why no `jdk_all` definition in this case?

Reply:
Tried not to introduce new `*_all` groups here. `jdk_all` would be the same as `jdk:all`, TBH. But we still can do it for symmetry reasons, see new commit.

Reply:
"all" looks okay but the comment "Catch-all" suggests something else, shouldn't be "All tests"?

Reply:
Yeah, we can do "All tests" instead. See new commit.